src/ui/map.js

```diff
@@ -276,39 +276,45 @@ class Map extends Camera {
     _requestManager: RequestManager;
 
     /**
-     * The map's {@link ScrollZoomHandler}, which implements zooming in and out with a scroll wheel or trackpad.
+     * The map's `ScrollZoomHandler`, which implements zooming in and out with a scroll wheel or trackpad.
+     * Find more details and examples using `scrollZoom` in the {@link ScrollZoomHandler} section.
      */
     scrollZoom: ScrollZoomHandler;
 
     /**
-     * The map's {@link BoxZoomHandler}, which implements zooming using a drag gesture with the Shift key pressed.
+     * The map's `BoxZoomHandler`, which implements zooming using a drag gesture with the Shift key pressed.
+     * Find more details and examples using `boxZoom` in the {@link BoxZoomHandler} section.
```

Comment:
I think having the @link at the very first mention of BoxZoomHandler is better. Anytime you first encounter a new technical term, having a link to further explain it is very helpful as a beginner. Moving the link further down makes it less likely I'd see and click the link.